app/configurations/waltti.js

```diff
@@ -41,17 +41,17 @@ export default {
 
   transportModes: {
```

Comment:
Since we are modding the config at runtime maybe we should not edit this file directly as it can create conflits later on or be over written by pulls / merges.